src/test/java/teammates/test/cases/testdriver/BackDoorTest.java

```diff
@@ -71,6 +72,33 @@ public void testDeletion() {
         assertEquals(Const.StatusCodes.BACKDOOR_STATUS_SUCCESS, status);
         verifyAbsentInDatastore(fq);
 
+        // ----------Deleting Feedback Session entities-------------------------
+
+        // #Session 1 in Course 1
+        FeedbackSessionAttributes feedbackSession1InCourse1 = dataBundle.feedbackSessions.get("session1InCourse1");
+        verifyPresentInDatastore(feedbackSession1InCourse1);
+        status = BackDoor.deleteFeedbackSession(feedbackSession1InCourse1.getFeedbackSessionName(),
+                feedbackSession1InCourse1.getCourseId());
+        assertEquals(Const.StatusCodes.BACKDOOR_STATUS_SUCCESS, status);
+        verifyAbsentInDatastore(feedbackSession1InCourse1);
+
+        // verify if question of session is deleted
+        FeedbackQuestionAttributes question1InSession1InCourse1 =
+                dataBundle.feedbackQuestions.get("qn1InSession1InCourse1");
+        verifyAbsentInDatastore(question1InSession1InCourse1);
```

Comment:
As discussed, the cascading is supposed to occur. I think in order to test this, can we do a `verifyPresentInDatastore(question1InSession1InCourse1)`, before we do the deletion of the session?

Reply:
Sure. Will add it